File: Linear_search/Python/linear_search.py
def linear_search(arr, target):
  for i in range(len(arr)):
    if arr[i] == target:
      return i
  return None

num = [1, 23, 15, 976, 234]
required = 976

res = linear_search(num, required)
if res != None:
  print(f'The element {required} was found at the {res} position')
else:
  print(f'The element {required} was not found')

File: Linear_search/Go/linear_search.go
package main
import "fmt"

func LinearSearch(arr []int, required int) int {
    for i := 0; i < len(arr); i++ {
        if arr[i] == required {
            return i
        }
    }
    return -1
}

func main() {
    num := []int{123, 53, 345, 12}
    required := 12

    res := LinearSearch(num, required)
    if res != -1 {
        fmt.Printf("The element %d found at position %d\n", required, res)
    } else {
        fmt.Printf("The element %d was not found", required)
    }
}